modules/users/client/controllers/authentication.client.controller.js

```diff
@@ -14,6 +14,11 @@
     vm.getPopoverMsg = PasswordValidator.getPopoverMsg;
     vm.signup = signup;
     vm.signin = signin;
+    vm.credentials = {};
+    vm.credentials.course = 'course1';
+    vm.credentials.roles = 'TA';
+    vm.credentials.UFID = 12345678;
+    vm.credentials.section = 'section1';
```

Comment:
We shouldn't hard code this at all despite my earlier comments. Since we have a sign up page that grabs these from the user, we need to not hard code this.